test/libsolidity/syntaxTests/constants/initialization/abi_encode_call_const_args.sol

```diff
@@ -0,0 +1,4 @@
+contract A {
+    function f() external {}
+    bytes constant fCallA = abi.encodeCall(A.f, ());
```

Comment:
I'd actually add arguments to the function as well:

```suggestion
    bool constant FLAG = true;
    function f(uint, bool) external {}
    bytes constant fCallA = abi.encodeCall(A.f, (123, FLAG));
```

Unless you want to make it a separate test - having a function without args covered does make sense too.